WordPress/src/main/java/org/wordpress/android/util/config/JetpackFeatureRemovalRemoteFields.kt

```diff
@@ -74,3 +74,16 @@ class PhaseSelfHostedPostLinkConfig @Inject constructor(appConfig: AppConfig) :
         appConfig,
         PHASE_SELF_HOSTED_BLOG_POST_LINK_REMOTE_FIELD
     )
+
+const val PHASE_FOUR_OVERLAY_FREQUENCY_IN_DAYS_REMOTE_FIELD = "phase_four_overlay_frequency_in_days"
+const val PHASE_FOUR_OVERLAY_FREQUENCY_IN_DAYS_DEFAULT_VALUE = "4"
```

Comment:
@AjeshRPai - Should the default be 0 or 4 ?

Reply:
Hey @zwarm My Bad, the default value should be 0. Fixed in [7d2b666](https://github.com/wordpress-mobile/WordPress-Android/pull/18934/commits/7d2b666885e09e34776e1be4551ea959f47fee89)